go.mod

```diff
@@ -56,6 +56,7 @@ require (
 	golang.org/x/tools v0.1.8
 	golang.zx2c4.com/wireguard v0.0.0-20211116201604-de7c702ace45
 	golang.zx2c4.com/wireguard/windows v0.4.10
+	gvisor.dev/gvisor v0.0.0-20220126021142-d8aa030b2591
 	honnef.co/go/tools v0.2.2
 	inet.af/netaddr v0.0.0-20211027220019-c74959edd3b6
 	inet.af/netstack v0.0.0-20211120045802-8aa80cf23d3c
```

Comment:
I'd have expected this to disappear.

Reply:
Why? That's the module name that netstack is in.

Reply:
Didn't we fully move to upstream? I thought the inet.af/netstack was the fork that we're removing

Reply:
run `go mod tidy`. these lines should disappear.

Reply:
oh, you're commenting on line 62. The highlight above made me think you were commenting on line 59. 🤦‍♂️

Reply:
The reason `go mod tidy` didn't clean it for me was because I had local test files not in git that were keeping `inet.af/netstack` alive. Fixed.